.github/workflows/windows.yml

```diff
@@ -0,0 +1,44 @@
+name: Windows Installer
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - master  
+
+env:
+  MSYS2_ROOT: C:\msys64
+
+jobs:
+  WindowsInstaller:
+    runs-on: windows-latest
```

Comment:
I typically prefer specific version than `*-latest` because `latest` is moving target. On other hand, `latest` allows to find issues with new releases.

Lets see if this become problem and then we can decide.

Reply:
Sure, we already have `windows-2019` working in Azure